esphome/components/i2c/__init__.py

```diff
@@ -43,6 +50,7 @@ def i2c_device_schema(default_address):
     """
     schema = {
         cv.GenerateID(CONF_I2C_ID): cv.use_id(I2CComponent),
+        cv.Optional(CONF_MULTIPLEXER): cv.Schema(I2CMULTIPLEXER_SCHEMA),
```

Comment:
I2CMULTIPLEXER_SCHEMA is already a schema
```suggestion
        cv.Optional(CONF_MULTIPLEXER): I2CMULTIPLEXER_SCHEMA,
```